update.sh

```diff
@@ -8,6 +8,7 @@ if [ ${#versions[@]} -eq 0 ]; then
 	versions=( */ )
 fi
 versions=( "${versions[@]%/}" )
+shaPage=$(curl -s 'https://www.ruby-lang.org/en/downloads/')
```

Comment:
I would think that is more pertinent in Dockerfiles, rather than update scripts.